### PR TITLE
Support for Query IncludeAll

### PR DESCRIPTION
**Pull request by dplewis**
Closes: https://github.com/parse-community/parse-server/issues/4644

Includes all pointers without having to use multiple `include()`.

Let me know if there is a better way to implement it.

### CODE REVIEW

File: spec/ParseQuery.spec.js [hunk 1]
```diff
@@ -3684,6 +3684,75 @@ describe('Parse.Query testing', () => {
     });
   });
 
+  it("includeAll", (done) => {
```
Comment (acinader) [inline spec/ParseQuery.spec.js]: note to self: add rule on " '.  Surprised the default airbnb we use doesn't have an opinion....

File: src/RestQuery.js [hunk 2]
```diff
@@ -552,6 +558,29 @@ RestQuery.prototype.runCount = function() {
     });
 };
 
+RestQuery.prototype.handleIncludeAll = function() {
```
Comment (acinader) [inline src/RestQuery.js]: function block comment would be nice....

File: src/RestQuery.js [hunk 3]
```diff
@@ -552,6 +558,29 @@ RestQuery.prototype.runCount = function() {
...
+      const keyFields = [];
+      for (const field in schema.fields) {
+        if (schema.fields[field].type && schema.fields[field].type === 'Pointer') {
+          includeFields.push([field]);
+          keyFields.push(field);
+        }
+      }
+      // Add fields to include, keys, remove dups
+      this.include = [...new Set([...this.include, ...includeFields])];
+      if (this.keys) {
```
Comment (acinader) [inline src/RestQuery.js]: comment like: 

// if `this.keys` not set, then all keys already included

assuming I've got that right.

**Comment by dplewis** [comment]
@acinader Added the changes

**Pull request merged**